UE/Tests/Application/ApplicationTestSuite.cpp

```diff
@@ -125,4 +125,19 @@ TEST_F(ApplicationConnectedTestSuite, shallHandleSms)
     objectUnderTest.handleSms(sms);
 }
 
+TEST_F(ApplicationConnectedTestSuite, shallHandleCallRequest)
+{
+    EXPECT_CALL(userPortMock, showCallRequest(_));
+    EXPECT_CALL(timerPortMock, startTimer(30000ms));
+    objectUnderTest.handleCallRequest(PHONE_NUMBER);
+}
+
+TEST_F(ApplicationConnectedTestSuite, shallHandleCallAccepted)
+{
+    EXPECT_CALL(btsPortMock, sendCallAccepted(_));
+    EXPECT_CALL(userPortMock, showTalking());
+    EXPECT_CALL(timerPortMock, stopTimer());
+    objectUnderTest.handleCallAccept(common::PhoneNumber{});
+}
+
```

Comment:
Myślę że będzie trzeba dodać nową strukturę "ApplicationTalkingTestSuite". W tym momencie chyba nie jest niezbędna, ale skoro przechodzisz do tego stanu to można też ją zrobić. Mam na myśli strukturę + konstruktor

Reply:
I don't think that it is crucial right now. Would rather merge this as fast as possible as long as the code is correct